cookiecutter.json

```
@@ -5,6 +5,12 @@
     "author_name": "",
     "author_email": "",
     "project_url": "",
+    "sourcecode_url": "",
+    "github_repo": "{{ cookiecutter.sourcecode_url | replace('https://github.com/', '') }}",
```

Here's a thought, why don't we swap these around?
```suggestion
    "github_repo": "sunpy/sunpy",
    "sourcecode_url": "{{ 'https://github.com/' + cookiecutter.github_repo if github_repo else '' }}",
```